content/redis/RedisReplication.aux

```diff
@@ -0,0 +1,69 @@
+\relax 
```

Comment:
is einfacher, wenn du des geschwind aus dem commit entfernst ;)
wenns nicht klappt, mach ichs nach dem mergen